src/messages/gateway.rs

```diff
@@ -95,7 +97,7 @@ impl Serialize for TaskState {
             TaskState::Invalid => "Invalid",
             TaskState::Waiting => "Waiting",
             TaskState::Finished => "Finished",
-            TaskState::Running => "Running",
+            TaskState::Running(_) => "Running",
```

Comment:
The serialization here needs to be changed. It's a more complex change, so we will probably do it in another PR. @spirali

Reply:
there can not be serialization changed everythink works well